MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
the .targets fixup sounds like a great idea.  

We should also include the .dll in the nuget for those people using it.